frontend/src/app/TelemetryService.ts

```diff
@@ -68,6 +73,42 @@ export function filterStaticFiles(envelope: ITelemetryItem) {
   }
 }
 
+export function filterUnneededItems(envelope: ITelemetryItem) {
+  const staticFileFound = filterStaticFiles(envelope) !== undefined;
+  if (staticFileFound) return false;
+
+  filterPotentialOktaRedirectEvent(envelope);
+}
+
+export function filterPotentialOktaRedirectEvent(envelope: ITelemetryItem) {
+  // Okta redirects only come from page views events
+  const eventIsPageView = envelope?.baseType === "PageviewData";
+  if (!eventIsPageView) return true;
```

Comment:
nit: I'm not sure how others feel, but making this function return an envelope or true feels weird to me. Maybe making this return the original envelope back?

Reply:
Oooo yeah I felt the same way. I actually think that the method filterStaticFiles should have been left unchanged because that is literally acting as a blacklist filtering for logs. And a new method maybe called sanitizeAuthToken()
should handle the new logic

Reply:
Agreed that this is weird. Originally I didn't have this, but when I went to test the function, I could think of a great way to spy on the envelop object within the test harness without spinning up mocking one level up at the application insights level, so I just returned it. If you know of a better way to do this though happy to change it!

Reply:
I didn't know how suggest commits like @BobanL so I ended up creating [this PR](https://github.com/CDCgov/prime-simplereport/pull/5489) I have not tested this @fzhao99 ... but I think the logic could be split along these lines

Reply:
sound good! thanks so much Johanna. I'll go back and fix the tests and then re-request reviews